routing/probability_apriori.go

```diff
@@ -169,7 +143,9 @@ func (p *AprioriEstimator) getNodeProbability(now time.Time,
 
 	// We reduce the apriori hop probability if the amount comes close to
 	// the capacity.
-	apriori := p.AprioriHopProbability * capacityFactor(amt, capacity)
+	apriori := p.AprioriHopProbability * capacityFactor(
+		amt, capacity, DefaultCapacityFraction,
```

Comment:
The `DefaultCapacityFraction` field should be based on the internal config (so it's configurable).

Reply:
I did that in the second commit, I could squash the first and second commit, just wanted to separate the functional change from the configuration change

Reply:
Realized that it may be better to have the configuration commit first and then apply changes to the capacity factor, that way it's nicer to see differences in the test.